src/test/resources/config/application-artemis.yml

```diff
@@ -48,6 +48,16 @@ artemis:
         vcs-credentials: fake-key
         artemis-authentication-token-key: fake-key
         artemis-authentication-token-value: fake-token
+        build-images:
+            JAVA: "ls1tum/artemis-maven-template:java17-2"
```

Comment:
It's a bit unfortunate that we have to duplicate the information here. Any chance to tell the tests to load this information from a common place?

Reply:
I'll have a look at it. Maybe this can simply be deleted.

Reply:
@krusche it seems that the properties must configured explicitly. Maybe someone has another idea to avoid this duplicates.
(See https://github.com/ls1intum/Artemis/pull/4455/commits/9e3fccd7374f7b13296660fe843f2ed2843746d7 & https://github.com/ls1intum/Artemis/pull/4455/commits/b4bc4ff0217cfdc3b33a3848695352e75c9107ce & test executions)

Reply:
Done